### DISCUSSION TITLE
Help, gh-pages not showing images

### DISCUSSION THREAD

**Discussion by 2-GO** [community/community#56768] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I published a sushi website project, but encountered an error with the images not working, I have tried to look at the pathway and everything looks to be in sequence and the images show up when I click them in the assets folder. I would appreciate any advice or suggestions, I'm not sure what else to do. Here is a link to the repository [https://github.com/2-GO/Sushiman](https://github.com/2-GO/Sushiman). Thank you.

**Comment by airtower-luna** [comment, marked as answer]
The problem is that the [`index.html` on your `gh-pages` branch](https://github.com/2-GO/Sushiman/blob/892a24b1554b4268ab0f73ad74c14c31251452df/index.html) uses absolute paths for about every resource it includes from your repository, starting with the icon:

```html
    <link rel="icon" type="image/png" href="/sushi.png" />
```

With this the browser will look for `https://2-go.github.io/sushi.png`, which doesn't exist: Your site gets deployed at `https://2-go.github.io/Sushiman/`, so the picture will end up at `https://2-go.github.io/Sushiman/sushi.png`. Same for the CSS and about every image.

The best fix is to remove the leading `/`, making the URL relative. Alternatively make sure to include the correct base path in the absolute paths.

**Comment by 2-GO** [comment]
Hi again, @airtower-luna. Thanks for explaining that fix to me. It worked. I have another question related to the same project, do you know why I cant see any text on the gh-branch published website?

**Reply by airtower-luna** [reply]
There seem to be some issues with the stylesheet, most text elements start out with `opacity: 0` (so, fully transparent).

Possibly the script your site includes is meant to make them fade in or something, but it has the same absolute path issue as your images did: https://github.com/2-GO/Sushiman/blob/24c7413f8e3e5a54e6da2eac825194a5fce89395/index.html#L10

**Reply by 2-GO** [reply]
ok, I will look into it, I appreciate it.
